src/main/java/nextstep/fp/StreamStudy.java

```diff
@@ -27,7 +27,18 @@ public static void printLongestWordTop100() throws IOException {
                 .get("src/main/resources/fp/war-and-peace.txt")), StandardCharsets.UTF_8);
         List<String> words = Arrays.asList(contents.split("[\\P{L}]+"));
 
-        // TODO 이 부분에 구현한다.
+        List<String> collections = words
+            .stream()
+            .filter(word -> word.length() > 12)
+            .sorted((word1, word2) -> Integer.compare(word2.length(), word1.length()))
+            .distinct()
+            .limit(100)
+            .map(word -> word.toLowerCase())
+            .collect(Collectors.toList());
```

Comment:
`.forEach()`를 통해서 곧바로 출력을 해볼수도 있겠습니다.
```suggestion
            .forEach(System.out::println);
```